src/_base/_twig/docker-compose.yml/application.yml.twig

```diff
@@ -74,6 +75,7 @@
       - traefik.enable=false
     networks:
       - private
+      - shared
```

Comment:
does this make talking to things like mysql problematic like nginx for php-fpm?

Reply:
is there any simple low cost (e.g. go-based scratch) tcp relay that could be used to solve this without having to recursively use more links?

Reply:
Multiple projects started at once with this change applied seem to be talking to the right places for me from the outside via traefik.

Inside the `console` container of one leads to this though, as you expect:
```
$ getent hosts nginx
172.19.0.3      nginx
172.19.0.8      nginx
172.19.0.19     nginx
172.19.0.7      nginx
172.19.0.14     nginx
```

Hmmm

Reply:
my expectations is php -> mysql would be round-robining the mysql service IP addresses, but not clever enough to pick up a different server with failing auth on the incorrect ones. Same for any other service we might need php to talk to, so I don't think we should introduce the links to every service that php talks to.
a container that tcp relays can be on the private + shared network while php sticks to private

Reply:
oh, actually, it wouldn't, as mysql isn't on the shared network

Reply:
it is annoying for php (like console) to nginx tho, but that's a different issue

Reply:
Now implemented as a nginx tcp proxy_pass between console/php-fpm talking to `mailhog-relay` in the private network and `mailhog` in the shared one.
How's that? :)

Reply:
I'd "resolved" the conversation as I realised it's unlikely a problem, due to other services generally not being on shared network.

If you think a solution still needed like you have done, then that's ok

Reply:
I suppose you mean it'll be a problem for php -> nginx

Reply:
Yeah, best all around to not join the shared network